hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -317,6 +317,27 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "DIT",
```

Comment:
@GregAC will you be updating this to roughly following the naming convention established so far?
we are using the form `ASSET`.`COUNTERMEASURE`.

Of course, not everything will neatly fit based on what we have defined so far, so we can certainly work on creating new categories.

Reply:
Yes I will do. I had just copied these names from the countermeasures sheet, hadn't realised a standardised form was required.

Should I update the names in the countermeasures sheet too?

Reply:
yes if you can, since @cdgori will use that as a reference when doing d2s reviews.
@rswarbrick also added the list of standardized names into our documentation, so if you need to enhance for your purposes, feel free to make changes there too.

Reply:
For the docs, see: https://docs.opentitan.org/doc/rm/comportability_specification/#countermeasures